mmdet/models/necks/fpn.py

```diff
@@ -93,7 +90,6 @@ def __init__(self,
         self.no_norm_on_lateral = no_norm_on_lateral
         self.fp16_enabled = False
         self.upsample_cfg = upsample_cfg.copy()
-        self.caffe2_xavier_init = caffe2_xavier_init
```

Comment:
We should keep this option and report a deprecated warning. Otherwise downstream repos like mmdet-projects will fail?

Reply:
I add this argument in `Reimplement AutoAssign #4295`, I am sure this arg only used in autoassign.

Reply:
OK great